values.yaml

```
@@ -211,6 +211,9 @@ alertmanager:
     annotations: {}
     labels: {}
 
+  serviceAccount:
+    name:
```

```suggestion
    # -- "" disables the individual serviceAccount and uses the global serviceAccount for that component
    name: ""
```

Small suggestion, I would prefer that than having no value at all. Looks odd to me